src/Evergreen/Migrate/V10.elm

```diff
@@ -73,8 +82,8 @@ toFrontend old =
 
 migrate_Types_BackendModel : Evergreen.V1.Types.BackendModel -> Evergreen.V10.Types.BackendModel
 migrate_Types_BackendModel old =
-    { users = Dict.empty
-    , authenticatedSessions = Dict.empty
+    { users = old.users |> Dict.map (\k -> migrate_Api_User_UserFull)
+    , authenticatedSessions = old.sessions
```

Comment:
I noticed you originally were nuking both users and sessions – if that was the intended behaviour then will need to restore, I wasn't sure so I left the migration behaviour.